packages/s3-presigned-post/src/createPresignedPost.ts

```diff
@@ -86,21 +86,18 @@ export const createPresignedPost = async (
   const signingKey = await getSigningKey(sha256, clientCredentials, shortDate, clientRegion, "s3");
   const signature = await hmac(sha256, signingKey, encodedPolicy);
 
-  let endpoint = await client.config?.endpoint?.();
-  let isEndpointV2 = false;
-
-  if (!endpoint) {
```

Comment:
previously, this would switch to endpointsV2 only if no custom v1 endpoint were provided.